### ISSUE TITLE
Add test for empty bold tags functionality

### ISSUE THREAD

**Issue by rexagod** [publiclab/PublicLab.Editor#202]
Hi, this is a [first-timers-only issue](https://publiclab.github.io/community-toolbox/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/publiclab.Editor/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### Problem 
To implement a test for detecting empty bold tags (for #198). This test would help automate builds determine whether the #198 functionality is working fine or not.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] Fork and clone this repository

- [ ] Create a new uniquely named branch and checkout to it

- [ ] Install dependencies using `npm install`
- [ ] Install grunt cli using `npm install -g grunt-cli`

### Implement the Solution

The corresponding test for this feature can be implemented by adding the following funtion to `./spec/javascripts/rich_text_module_spec.js` file.

````js
it("displays alert for empty bold tags",function() {

    var enter = jQuery.Event("keydown", {keyCode:13})

    var temp_el = module.textarea

    module.setMode('markdown');

    module.textarea.innerHTML = '**';

    $(temp_el).trigger(enter)

    module.textarea.innerHTML = '**';

    $(temp_el).trigger(enter)

    expect($('.invalid-bold-tags-warning').length).not.toBeNull();

  })
````
- [ ] Build using `grunt build`

- [ ] Test your assertions using `grunt jasmine`. If all specs pass, follow next steps.

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### 🤔❓ Questions?

Leave a comment below!

### Is someone else already working on this?

We encourage you to link to this issue by mentioning the issue # in your pull request, so we can see if someone's already started on it. **If someone seem stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://publiclab.github.io/community-toolbox/#r=all). Thanks!

**Comment by rexagod** [comment]
@publiclab/reviewers Think we can label this as a `fto`? If not, what changes should I make? Looking forward to opening more `fto`'s and your guidance will be appreciated! Thanks!

**Comment by jywarren** [comment]
Hi, @rexagod thanks! Would you mind taking a close look at this guide, which helps formatting of FTOs? This will be very awesome once it's fully formatted! Eager to build this community 👍 🎉 🙌 

https://publiclab.org/n/13667

**Comment by rexagod** [comment]
I've edited the issue. Can we now label it as an `fto`?

**Comment by jywarren** [comment]
Hi, @rexagod -- we're almost there! Sorry i took time off over the weekend, just chiming back in now. Actually it can be really effective to add the "preamble" as in this example, with some extra welcoming friendly language: https://github.com/publiclab/plots2/issues/4189

Once we add that, we should be good to go! Thank you and i'm updating the activity guide I'd linked to to emphasize that section as well! (FTOs have evolved over the years 😄 )

**Comment by gauravano** [comment]
Hi @rexagod I transformed it to FTO, you can use this template for create new issue. It's in reference to Publiclab editor.

**Comment by rexagod** [comment]
Thanks! I've made the required edits.

**Comment by IgorWilbert** [comment]
Hello! I need to practice using Git for cloning and forking, since I made a big mess last time I tried. May I take this issue please?

**Comment by gauravano** [comment]
Go ahead @IgorWilbert! Thanks!

**Comment by jywarren** [comment]
Awesome!!!

**Comment by gauravano** [comment]
I see this issue is solved in #220 so closing this issue. @IgorWilbert a suggestion - Including Fixes #[issue-number] in the PR, closes the issue after PR gets merged, so it's a good practice. Thanks!
